tests/run_tests.py

```diff
@@ -110,7 +110,7 @@ def test():
     parser.add_argument('-c',
                         '--cpp',
                         dest='cpp',
-                        default=True,
+                        default=False,
```

Comment:
why disable cpp test? ;)

Reply:
It's interesting that with `action='store_true'` + `default=True`, it becomes impossible to turn it off :p